backend/index.js

```diff
@@ -1,6 +1,7 @@
 const express = require('express');
 const bodyParser = require('body-parser');
 const signUpRoute = require('./routes/auth/sign-up');
+const loginRoute = require('./routes/login');
```

Comment:
Почему бы не использовать такой же маршрут, как и у регистрации?
```suggestion
const loginRoute = require('./routes/auth/login');
```

Reply:
Ok